packaging/ubuntu-16.04/not-installed

```diff
@@ -0,0 +1 @@
+debian/tmp/usr/bin/uboot-go
```

Comment:
I didn't know about `not-installed`. Assuming it flags things to not warn about not installing, we might be able to drop a couple of `rm`'s from the `override_dh_install` rule